cutgeneratingfunctionology/spam/basic_semialgebraic.py

```diff
@@ -29,6 +31,10 @@
     # old Sage
     from sage.libs.ppl import Variable, Constraint, Linear_Expression, Constraint_System, NNC_Polyhedron, Poly_Con_Relation, Poly_Gen_Relation, Generator, MIP_Problem, point as ppl_point
 
+from pplite import Variable as pplite_Var, Constraint as pplite_Con, Linear_Expression as pplite_Lin_Expr, Affine_Expression as pplite_Aff_expr, NNC_Polyhedron as pplite_NNC_Polyhedron, PPliteGenerator, Polyhedron_Constraint_Rel, Polyhedron_Generator_Rel
```

Comment:
this shouldn't be imported unconditionally because we don't want to make pplite a mandatory dependency.

see below -- best put this new pplite material in a separate file